module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,17 @@
+#include <algorithm>
+
+#include "sort.hpp"
+
+bool sortWords(std::string a, std::string b) {
```

Comment:
Rename a function to express appropriate behaviour of function. In my opinion current name is misleading because function makes **comparasion** not **sorting**.

Reply:
For me it's not. We are defining what criteria we are using during sorting.

Reply:
For me, it is ;) I agree with @Rosein.
I would expect any sort* function to return `void` or the sorted container. 
You don't need to change it now, but keep it in mind in future.